docs/source/production.rst

```diff
@@ -14,7 +14,7 @@ a non-zero code will be returned which can be interrogated to find out
 more.
 
 - At the moment all error states related to linting return *65*.
-- An error as a result of a sqlflSQLFluffuff internal error will return *1*.
+- An error as a result of a SQLFluff internal error will return *1*.
```

Comment:
Whoops, I bet this made it through in #616 good catch 🧤 ⚾

Reply:
This was actually me like an hour ago! Ooops. Missed my double-click!